client/extension.js

```diff
@@ -79,6 +80,13 @@ function activate(context) {
 		})
 	);
 
+	vscode.workspace.onDidChangeConfiguration(() => {
```

Comment:
I don't believe we need this at all. This is handled on the server. Any open text document will re-lint on any configuration change. - please remove this event handler and the JSON stringify stuff.

Reply:
hmm I guess - maybe I am wrong here, because it needs to now pick up the new files?.. I don't like the idea of reloading the window here, there must be a better way. Let me know the use case I am missing if you feel this needs to stay!

Reply:
after a little playing and some further thought, I think reloading the window is an acceptable first step.

Reply:
Yeah, I played around with it a bit, and I'm not sure the best way to handle this either. I tried updating the configuration and restarting the client with `client.restart()` but that seems to end with an error. Unfortunately, my work with vscode extensions are limited.